src/server/api/terrain.js

```diff
@@ -90,20 +94,19 @@ export const call = (
         apiURL.search = querystring.stringify(query);
     }
 
-    logger.info(`TERRAIN ${userID} ${method} ${apiURL.href}`);
+    logger.info(
+        `TERRAIN ${userID} ${method} ${apiURL.href} ${JSON.stringify(data)}`
```

Comment:
I added the request body to the log, but we don't need it if others think this will be noisy or is unnecessary

Reply:
Logging request body is helpful 👍🏽

Reply:
Let's try it out for a while, we can always revert it if it gets too noisy 👍